crypto/blake2/TODO

```diff
@@ -0,0 +1,4 @@
+The code uses NATIVE_LITTLE_ENDIAN in #if statements.  Is this right?
```

Comment:
Yes, L_ENDIAN is defined in CFLAGS.  But also yes, prefer to do order-neutral code first.